static/css/styles.css

```diff
@@ -351,22 +318,137 @@ aside#overlay.hide {
     }
 }
 
-.navbar-toggler {
-    color: white;
+/* Page Settings */
+@page { 
+    margin: 2cm;
 }
 
-.navbar-toggler .fa-solid {
-    color: white;
+/* Paragraph Styles */
+P { 
+    margin-bottom: 0.21cm;
+}
+
+P.western { 
+    so-language: en-GB;
+}
+
+/* General Styles */
+body {
+    font-family: 'Lato', sans-serif;
+    margin: 0;
+    padding: 0;
+    background-color: #f5f5f5;
+    color: #333;
+}
+
+/* Primary Headers and Titles */
+h1, h2, h3, h4, h5, h6 {
+    font-family: 'Montserrat', sans-serif;
+}
+
+h2 {
+    font-size: 2em;
+    font-weight: bolder;
+    margin-bottom: 0.5em;
+}
+
+h3 {
+    font-size: 1.75em;
+    font-weight: bold;
+    margin-top: 1em;
+    margin-bottom: 0.5em;
+}
+
+h4 {
+    font-size: 1.5em;
+    font-weight: bold;
+    margin-top: 1em;
+    margin-bottom: 0.5em;
+}
+
+/* Secondary Text */
+p, ul, li {
+    font-family: 'Lato', sans-serif;
+    font-size: 1em;
+    margin-bottom: 1em;
+}
+
+ul {
+    list-style-type: none;
+    padding: 0;
 }
 
-@page { margin: 2cm }
-P { margin-bottom: 0.21cm }
-P.western { so-language: en-GB }
-H3 { margin-bottom: 0.21cm }
-H3.western { so-language: en-GB }
-H3.cjk { font-family: "SimSun" }
-H3.ctl { font-family: "Arial" }
-H4 { margin-bottom: 0.21cm }
-H4.western { so-language: en-GB }
-H4.cjk { font-family: "SimSun" }
-H4.ctl { font-family: "Arial" }
+li {
+    font-size: 1em;
+    margin-bottom: 0.5em;
+}
+
+/* Callout Section Styles */
+#callout {
+    background-color: white;
+    padding: 2em 0;
+}
+
+.callout-pic {
+    max-width: 100%;
+    height: auto;
+}
+
+/* Container and Row Styles */

```

Comment:
Do not override the Bootstrap classes! Please remove all of this new code that overwrites the Bootstrap framework.